infrastructure/restapi/src/main/java/tech/pegasys/teku/infrastructure/restapi/RestApiBuilder.java

```diff
@@ -118,11 +119,11 @@ public RestApi build() {
 
     addExceptionHandlers(app);
 
+    Optional<String> restApiDocs = Optional.empty();
     if (openApiDocsEnabled) {
-      final String openApiJson = openApiDocBuilder.build();
-      app.get("/swagger-docs", ctx -> ctx.json(openApiJson));
```

Comment:
I'd be very tempted to keep registering the `/swagger-docs` endpoint here since every other endpoint is registered here.  And that way `RestApi` isn't performing work in the constructor which tends to make testing harder.